frontend/src/css/more-info.css

```
@@ -123,27 +118,27 @@
         background-color: rgba(0, 0, 0, 0.3);
         border-radius: 100rem;
-        font-size: 0.8rem;
     }
 }
 
 .more-info--like-asset,
 .more-info--hide-asset,
 .more-info--webhook,
 .more-info--image-link {
+    padding: 0.5rem 2rem;
     font-family: "Barlow", Arial, Helvetica, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Barlow" (font-family-name-quotes)

(font-family-name-quotes)